trpc/queries/projectV2.ts

```diff
@@ -103,6 +103,30 @@ export const getChildrenProjectsId = (params: { in: Array<{ id: string }> }) =>
         );
 };
 
+export const getDeepChildrenProjectsId = (params: { in: Array<{ id: string }> }) => {
+    return db
+        .withRecursive('childrenTree', (qb) =>
+            qb
+                .selectFrom('_parentChildren')
+                .select('_parentChildren.B as id')
+                .$if(params.in.length > 0, (qb) =>
+                    qb.where(
+                        '_parentChildren.A',
+                        'in',
+                        params.in.map(({ id }) => id),
+                    ),
+                )
+                .union((qb) =>
+                    qb
+                        .selectFrom('_parentChildren')
+                        .select('_parentChildren.B as id')
+                        .innerJoin('childrenTree', 'childrenTree.id', 'A'),
+                ),
+        )
+        .selectFrom('childrenTree')
+        .selectAll();
```

Comment:
Maybe `select(['id'])`? We don't need anything else here

Reply:
it only has an identifier, why specify it explicitly?